test/command.js

```diff
@@ -85,3 +85,88 @@ test('execaCommandSync()', t => {
 	const {stdout} = execaCommandSync('node test/fixtures/echo.js foo bar');
 	t.is(stdout, 'foo\nbar');
 });
+
+test('$', async t => {
+	const {stdout} = await $`node test/fixtures/echo.js foo bar`;
+	t.is(stdout, 'foo\nbar');
+});
+
+test('$ accepts options', async t => {
+	const {stdout} = await $({shell: true, env: {TEST: 'test'}})`echo $TEST`;
```

Comment:
I believe this might not work on `cmd.exe`? This is the shell used by `shell: true` on Windows, and environment variables are using [a different syntax `%TEST%`](https://github.com/ehmicky/cross-platform-node-guide/blob/main/docs/4_terminal/environment_variables.md#referencing).

What about using the `stripFinalNewline` option instead, and checking to see if the newline is stripped?

Reply:
Another great catch! Updated the tests to use the `stripFinalNewline` option: https://github.com/sindresorhus/execa/pull/510/commits/bddf3179e26a50aff59c9eec72d703bf55d9eb3a